save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/OrganizationView.kt

```diff
@@ -219,7 +219,7 @@ class OrganizationView : AbstractView<OrganizationProps, OrganizationViewState>(
             confirmationType = ConfirmationType.DELETE_CONFIRM
             isConfirmWindowOpen = true
             confirmLabel = ""
-            confirmMessage = "Are you sure you want to delete this organization?"
+            confirmMessage = "Are you sure you want to delete organization ${props.organizationName}?"
         }
     }
```

Comment:
You should encapsulate all the deletion logic inside of new FC. This method should not exist.

Reply:
Done